### PR TITLE
fix: add canvas mouseout event

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* asis
asis

* tobe
tobe

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/chart/src/charts/chart.ts [hunk 1]
```diff
@@ -338,6 +323,27 @@ export default abstract class Chart<T extends Options> {
...
+    const canvas = this.painter.ctx.canvas;
+    const canvasRect = canvas.getBoundingClientRect();
+    // Calculate scale for chart affected by a CSS transform.
+    const scaleX = canvasRect.width / canvas.offsetWidth;
+    const scaleY = canvasRect.height / canvas.offsetHeight;
+
+    const mousePosition = {
+      x: (clientX - canvasRect.left) / scaleX,
+      y: (clientY - canvasRect.top) / scaleY,
+    };
```
Comment (jungeun-cho) [inline apps/chart/src/charts/chart.ts:338]: canvasRect 구조분해할당해서 바로 사용하는거어떄요?
`const { width, height, left, top } = canvas.getBoundingClientRect();`

File: apps/chart/src/charts/chart.ts [hunk 2]
```diff
@@ -338,6 +323,27 @@ export default abstract class Chart<T extends Options> {
...
+
+    if (eventType === 'mousemove' || eventType === 'mouseout') {
+      this.handleCanvasMouseEvent(eventType, mousePosition);
+    }
+
+    this.handleResponderEvent(event, mousePosition);
+  }
```
Comment (jungeun-cho) [inline apps/chart/src/charts/chart.ts:345]: 깔끔해졌어요~!

**Review by jungeun-cho — APPROVED** [review]
리뷰완료합니다~!

**Review by js87zz — APPROVED** [review]
고생하셨어요~!

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Pull request merged**